src/targets/franklin.jl

```diff
@@ -132,6 +152,7 @@ function generate_citation(note::FranklinNote; citations="")
 
     if isempty(citations)
 
```

Comment:
Could you add this into the docstring for this function? 

```julia
"""
FIXME: This is currently hardcoded for my own personal website. We need to adjust this to not be that way.
"""
```

I think we should open a separate issue to fix this.